### PR TITLE
Add meta description tags to pages for SEO

### LINKED ISSUE #532: Add meta descriptions to all pages
I'm working on adding meta descriptions to all pages as a part of SEO improvement and GCI.

Task: https://codein.withgoogle.com/tasks/6178373986942976/?sp-organization=6302639764013056&sp-is_beginner=False

### PR DESCRIPTION
**Pull request by soumitradev**
I have added meta descriptions tags to every HTML file on this website. I have omitted around 7-8 HTML files since they are either utf8, Google Analytics support scripts, or just the navigation bar on the homepage.

This resolves #532 an a part of #501.

### CODE REVIEW

**Comment by soumitradev** [comment]
I have built a preview here: https://julia-preview.netlify.com/, and it seems that the [Google Lighthouse](https://developers.google.com/web/tools/lighthouse) SEO score of the website stays at 90, but the Lighthouse tool doesn't give us the warning to add meta descriptions anymore.

**Comment by soumitradev** [comment]
On using the Lighthouse tool, I found that adding the Meta Descriptions does not change the SEO score of the website (I think this is because Google says that the Meta Description tags don't directly affect the SEO, but they do affect the SEO indirectly since people are more likely to click on links with good descriptions. This only affects click-through rate (CTR), which affects the SEO.)

Source for this information: https://webmasters.googleblog.com/2009/09/google-does-not-use-keywords-meta-tag.html

Quote from source:
"Even though we sometimes use the description meta tag for the snippets we show, we still don't use the description meta tag in our ranking."

They did say that they rarely use the description, but that seems a bit odd since their tool itself suggests adding the description tags.

**Comment by ViralBShah** [comment]
Leaving open for review for a day and will merge then.

File: _includes/benchmarks.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <!-- generated by JuliaLang/Microbenchmarks/bin/table.jl -->
 
+<head>
+  <meta name="description" content="These are benchmarks of Julia as compared to other popular programming langauges in terms of popular mathematical algorithms. Learn more here."/>
```
Comment (logankilpatrick) [inline _includes/benchmarks.html]: "benchmarks of Julia as compared to" should be "benchmarks of Julia compared to"

File: _includes/blogs.html [hunk 2]
```diff
@@ -1,4 +1,9 @@
 <!-- blog -->
+
+<head>
+  <meta name="description" content="Want to know more about the Julia blog? Click here to read the latest developments in the Julia Programming Language."/>
```
Comment (logankilpatrick) [inline _includes/blogs.html]: "read the latest developments in the Julia Programming Language" should maybe be "read the latest developments about the Julia Programming Language"

File: downloads/plotting.html [hunk 3]
```diff
@@ -3,6 +3,10 @@
 title:  Plotting in Julia
 ---
 
+<head>
+  <meta name="description" content="Get started with plotting packages for Julia today. Learn more about plotting in Julia with these powerful external packages here."/>
```
Comment (logankilpatrick) [inline downloads/plotting.html]: wording seems weird imo

Comment (soumitradev) [reply downloads/plotting.html]: Hmmm... I'm not all that good at writing descriptions. Maybe someone else can do this one?

**Comment by ViralBShah** [comment]
@logankilpatrick I think you can "suggest" the changes directly here, and they can just be accepted.

**Comment by logankilpatrick** [comment]
Yeah, I am traveling now for the day but I Will make the changes tonight unless this has already been merged

File: downloads/index.html [hunk 4]
```diff
@@ -3,6 +3,10 @@
 title:  Julia Downloads
 ---
 
+<head>
+  <meta name="description" content="These are the official latest binaries of Julia along with the instructions to download, install, and configure on MacOS, Windows, and Linux. Download here."/>
```
Comment (fungktt) [inline downloads/index.html]:
```suggestion
  <meta name="description" content="These are the official latest binaries of Julia along with the instructions to download, install, and configure on macOS, Windows, and Linux. Download here."/>
```

**Comment by ViralBShah** [comment]
I'll not merge. Merge when you think it is good to go.

**Comment by soumitradev** [comment]
Can confirm that the descriptions are working.

**Comment by ViralBShah** [comment]
Nice!

**Pull request merged**